lucet-module/Cargo.toml

```diff
@@ -11,7 +11,7 @@ edition = "2018"
 
 [dependencies]
 anyhow = "1.0"
-cranelift-entity = { path = "../wasmtime/cranelift/entity", version = "0.67.0" }
+cranelift-entity = { path = "../wasmtime/cranelift/entity" }
```

Comment:
afaik, we need to leave these `version` indicators in the tree for the sake of publishing to crates.io (which we rarely do, but dont want to box ourselves out of). I know its a big pain to keep them synced up, but I don't know how to publish without them there - maybe there is some other fix for that?

Reply:
Ah I figured that publication to crates.io wasn't on the table, but if that's desired I'll add these back in and update them all

Reply:
yeah :/ the last time we got around to publishing was back in February, which is pretty embarrassing. We use git, not the crate version, to manage this dependency internally, so publishing is never on our mind.